python/semantic_kernel/connectors/ai/open_ai/models/chat/tool_calls.py

```diff
@@ -0,0 +1,18 @@
+"""Class to hold chat messages."""
+from typing import Literal, Optional
+
+from semantic_kernel.connectors.ai.open_ai.models.chat.function_call import FunctionCall
+from semantic_kernel.sk_pydantic import SKBaseModel
+
+
+class ToolCall(SKBaseModel):
+    """Class to hold a tool call response."""
+
+    id: Optional[str] = None
+    type: Optional[Literal["function"]] = "function"
```

Comment:
I would prefer not to create a single value enum now and since this will be mostly filled by the response handling code a enum is not that usefull, literal works fine.